console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/widget/DateRangeSelector.java

```diff
@@ -329,7 +330,8 @@ public String validate(Field<?> field, String value) {
                 return null;
             }
         });
-
+        endDateField.setMaxLength(10);
+        startDateField.setMaxLength(10);
```

Comment:
Why are you setting this again, it was set already?

Reply:
@muros-ct Unnecessary lines removed.